src/sentry/tasks/data_export.py

```diff
@@ -18,17 +20,21 @@ class DataExportError(Exception):
 
 
 @instrumented_task(name="sentry.tasks.data_export.assemble_download", queue="data_export")
-def assemble_download(data_export):
+def assemble_download(data_export_id):
+    # Extract the ExportedData object
+    try:
+        data_export = ExportedData.objects.get(id=data_export_id)
+    except ExportedData.DoesNotExist as error:
+        return sentry_sdk.capture_exception(error)
```

Comment:
@leeandher Most places in the backend are importing `capture_exception` from `sentry.utils.sdk` and not `sentry_sdk`

Reply:
done!